embed/etcd.go

```diff
@@ -343,6 +350,10 @@ func (e *Etcd) serve() (err error) {
 }
 
 func (e *Etcd) errHandler(err error) {
+	if transport.IsClosedConnError(err) || err == grpc.ErrServerStopped {
```

Comment:
why is this necessary? shouldn't `stopc` be closed before calling etcdserver.Stop?